service-print/src/main/java/org/oskari/print/loader/CommandLoadImageWMS.java

```diff
@@ -43,7 +52,22 @@ public BufferedImage run() throws Exception {
                 .format(FORMAT)
                 .transparent(true)
                 .toKVP();
-        return CommandLoadImageFromURL.loadImageFromURL(new URL(request));
+        
+        LOG.info(request);
+        URL url = new URL(request);
+        for (int i = 0; i < RETRY_COUNT - 1; i++) {
+            try {
+                return ImageIO.read(url);
+            } catch (IOException e) {
+                LOG.warn(e);
```

Comment:
Perhaps this could wait a while after failure before retrying? Not too long since the user is waiting, but maybe ~50-100ms?

Reply:
Add some message for the warning and perhaps log only the last retry to prevent spam on the logs.